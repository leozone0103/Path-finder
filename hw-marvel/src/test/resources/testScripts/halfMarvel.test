# test relatively large input

LoadGraph heroes halfMarvel.csv

FindPath heroes name1 name3
FindPath heroes name3 name1
FindPath heroes name1 name2
FindPath heroes name2 name1
FindPath heroes name2 name3
FindPath heroes name3 name2
FindPath heroes name1 name1
FindPath heroes name2 name2
FindPath heroes name3 name3
FindPath heroes name1 name4
FindPath heroes name4 name1
FindPath heroes name4 name3