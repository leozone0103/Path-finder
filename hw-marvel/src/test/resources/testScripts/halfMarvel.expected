# test relatively large input

loaded graph heroes

path from name1 to name3:
name1 to name3 via b1
path from name3 to name1:
name3 to name1 via b1
path from name1 to name2:
name1 to name2 via b6
path from name2 to name1:
name2 to name1 via b6
path from name2 to name3:
name2 to name3 via b4
path from name3 to name2:
name3 to name2 via b4
path from name1 to name1:
path from name2 to name2:
path from name3 to name3:
path from name1 to name4:
no path found
path from name4 to name1:
no path found
path from name4 to name3:
no path found